webapp/pages/login.vue

```diff
@@ -7,19 +7,26 @@
 <script>
 import LoginForm from '~/components/LoginForm/LoginForm.vue'
 import { VERSION } from '~/constants/terms-and-conditions-version.js'
+import { mapGetters } from 'vuex'
 
 export default {
   layout: 'no-header',
   components: {
     LoginForm,
   },
+  computed: {
+    ...mapGetters({
+      user: 'auth/user',
+    }),
+  },
   asyncData({ store, redirect }) {
     if (store.getters['auth/user'].termsAndConditionsAgreedVersion === VERSION) {
       redirect('/')
     }
   },
   methods: {
-    handleSuccess() {
+    handleSuccess() {      
+      this.$i18n.set(this.user.locale)   
```

Comment:
What about a default here?
```suggestion
      this.$i18n.set(this.user.locale || 'en')   
```

Reply:
@ogerly can you please add this?